cloudferrylib/utils/drivers/ssh_chunks.py

```diff
@@ -45,7 +45,11 @@ def splitter(total_size, block_size):
 
 
 def remote_file_size(runner, path):
-    return int(runner.run('stat --printf="%s" {path}'.format(path=path)))
+    res = runner.run('stat --printf="%s" {path}'.format(path=path))
+    if res is not None:
+        return int(res)
+    else:
+        raise RuntimeError("Can not get information about file")
```

Comment:
This is not needed. If value returned by `stat` cannot be converted to `int` – it means there's something completely wrong here.